### DISCUSSION TITLE
Just a little compliment: I'm impressed so far 🤩

### DISCUSSION THREAD

**Discussion by binarykitchen** [web-infra-dev/rsbuild#3438] [General]
After my first attempts migrating from webpack, I must admit, I don't miss webpack any more (sorry webpack, you served me well for many years but ...)

Keep going with the byte dance team +1

![gif](https://i.giphy.com/media/v1.Y2lkPTc5MGI3NjExeXZ3ZTM2NDl2MWF0dnVtMm05bXdwc29paHJ4MDY0anE3bTl4OG5kaSZlcD12MV9pbnRlcm5hbF9naWZfYnlfaWQmY3Q9Zw/LOQyoLIojnizS949is/giphy.gif)

**Comment by chenjiahan** [comment]
Don't feel guilty about webpack; Rspack is a continuation of webpack, which proves the vitality of webpack. 😄

**Reply by binarykitchen** [reply]
True. That's a good response politicians should learn from ;)
